scripts/kind-e2e/lib_operator_deploy_subm.sh

```diff
@@ -25,7 +25,7 @@ GOPATH=$HOME/go
 subm_op_dir=$GOPATH/src/github.com/submariner-operator/submariner-operator
 mkdir -p $subm_op_dir
 
-cp -r ../operators/go/submariner-operator/ $subm_op_dir/..
```

Comment:
I think this is causing failures in later code. For example, I see:

kind-e2e/lib_operator_deploy_subm.sh: line 93: deploy/crds/submariner_endpoints_crd.yaml: No such file or directory

Reply:
For me it doesn’t work without this ;-).

Reply:
I went down a bit of a rabbit hole making the paths used in various places easier to understand, and I think also properly portable. So this done I think, but done a different way.